components/Chat/ChatNullState.tsx

```diff
@@ -68,7 +60,7 @@ const ChatNullState: React.FC<ChatNullStateProps> = ({
 
   return (
     <View style={styles.container}>
-      {!hasUserDismissedBanner && (
+      {/* {!hasUserDismissedBanner && (
```

Comment:
Commented out as we may want to include a similar link soon, just not the alpha group chat one